app/src/androidTest/java/com/minimalToDo/tests/CreateItem.java

```diff
@@ -0,0 +1,50 @@
+package com.minimalToDo.tests;
+
+import com.example.avjindersinghsekhon.minimaltodo.R;
+import com.minimalToDo.constants.Timeouts;
+import com.minimalToDo.screens.AddNewItem;
+import com.minimalToDo.screens.Home;
+import com.minimalToDo.screens.ToDoList;
+
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Created by valentin.boca on 2/9/2018.
+ */
+
+public class CreateItem extends EspressoTestBase {
+    @Test(timeout = Timeouts.TEST_TIMEOUT_SHORT)
+    public void testAddNewItemWithoutReminder() throws Exception {
+        AddNewItem.createNewItem( "fisier1" );
+        AddNewItem.clickCreate();
+        Helpers.checkIfUIObjectIsVisible( allOf( withText( "Fisier1" ), isCompletelyDisplayed() ), 3 );
```

Comment:
Aici ar trebui ceva de genul:
1. Adaugat metoda (daca nu este):

public static boolean isItemDisplayed(String text) throws Exception {
  return Helpers.checkIfUIObjectIsVisible(allOf(withText("text"), isCompletelyDisplayed()), 3);
}

2. O apelezi in test, in loc de linia 27:
AddNewItem.isItemDisplayed("Fisier1");